RushHour/RushHourMod.cs

```diff
@@ -22,13 +22,17 @@ public void OnSettingsUI(UIHelperBase helper)
                 new OptionsCheckbox() { readableName = "EXPERIMENTAL: Use improved commercial demand", value = false, uniqueName = "UseImprovedCommercial" }
             };
 
+            loadSettingsFromSaveFile();
```

Comment:
Ahhh, I see what's going on now. You're right in saying `loadSettingsFromSaveFile()` should be in another place, but this will do for now. With regards to `CimToolsHandler.CimToolBase.XMLFileOptions.Load()`, this only loads it into its internal storage, and I had that there when I was using another method to get values from it which I had since deleted and replaced with the `XMLFileOptions.Data.GetValue` code segments.

This wouldn't pull anything out unless you visited the settings though, so you're right in adding that method call in there. Thanks.